webpack.config.js

```diff
@@ -0,0 +1,57 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    library: 'AFRAME',
+    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/dist/',
+    filename: 'aframe-master.js'
+  },
+  devtool: 'source-map',
+  mode: 'development',
+  devServer: {
+    host: 'local-ip',
+    port: process.env.PORT || 9000,
+    hot: false,
+    liveReload: true,
+    static: {
+      directory: 'examples'
+    }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.INSPECTOR_VERSION': JSON.stringify(
+        process.env.INSPECTOR_VERSION
+      )
+    }),
```

Comment:
If you want to know more about DefinePlugin: https://webpack.js.org/plugins/define-plugin/